src/test/java/com/google/cloud/tools/managedcloudsdk/internal/install/LatestInstallerTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.internal.install;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+/** Tests for {@link LatestInstaller} */
+public class LatestInstallerTest {
+
+  @Mock private InstallScriptProvider mockInstallScriptProvider;
+  @Mock private ProcessBuilderFactory mockProcessBuilderFactory;
+  @Mock private ProcessBuilder mockProcessBuilder;
+  @Mock private Process mockProcess;
+  @Mock private InputStream mockInputStream;
+
+  @Rule public TemporaryFolder tmp = new TemporaryFolder();
+
+  @Before
+  public void setup() throws IOException {
+    MockitoAnnotations.initMocks(this);
+
+    Mockito.when(mockInstallScriptProvider.getScriptCommandLine())
+        .thenReturn(Arrays.asList("scriptexec", "test-install.script"));
+    Mockito.when(mockProcessBuilderFactory.newProcessBuilder()).thenReturn(mockProcessBuilder);
+    Mockito.when(mockProcessBuilder.start()).thenReturn(mockProcess);
+    Mockito.when(mockProcess.getInputStream()).thenReturn(mockInputStream);
+    Mockito.when(mockProcess.getErrorStream()).thenReturn(mockInputStream);
+    Mockito.when(mockProcess.exitValue()).thenReturn(0);
+  }
+
+  @Test
+  public void testCall() throws Exception {
+    Installer installer =
+        new LatestInstaller<>(
+            tmp.getRoot().toPath(),
+            mockInstallScriptProvider,
+            false,
+            null,
+            mockProcessBuilderFactory);
+    Path returnedSdkRoot = installer.call();
+
+    Mockito.verify(mockProcessBuilder, Mockito.times(1)).command(getExpectedCommand(false));
```

Comment:
According to the Mockito documentation, `times(1)` is used by default for `verify`.

http://static.javadoc.io/org.mockito/mockito-core/2.10.0/org/mockito/Mockito.html#4